testing/pac-ts-eks/tests/index.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright 2016-2020, Pulumi Corporation.  All rights reserved.
+
+import * as aws from "@pulumi/aws";
+import * as policy from "@pulumi/policy";
+import * as pulumi from "@pulumi/pulumi";
+
+const stackPolicy: policy.StackValidationPolicy = {
+    name: "eks-test",
+    description: "EKS integration tests.",
+    enforcementLevel: "mandatory",
+    validateStack: async (args, reportViolation) => {
+        const clusterResource = args.resources.find(r => r.isType(aws.eks.Cluster));
```

Comment:
Would it make sense to validate that there's only one cluster? Otherwise, other clusters would bypass all these checks.

Reply:
Done in #623 and applies to the guide